remove CEIP doc

kcoriordan opened the pull request:
closes: #1085 
Doc: remove references to (Customer Experience Improvement Program) CEIP

joshrosso reviewed (changes requested):
@kcoriordan thanks for getting this knocked out! Two pieces of feedback.

1. Can you squash and re-write your commit message to be more descriptive? We're trying to get better as we go to GA that the commit messages actually provide context on what was done.
    - we're really bad at this in engineering so hold us accountable too 😛.
1. It looks like you may have deleted `docs/site/content/docs/latest/deploy-ui.md` in this commit. Does that have anything to do with removing the CEIP doc? Maybe an accidentally addition?

jpmcb reviewed (changes requested):
This will be blocked on us bringing our Tanzu Framework dependency up to date with the changes to remove CEIP from the UI which will be after the v0.7.0 release
- https://github.com/vmware-tanzu/tce/issues/1273
- https://github.com/vmware-tanzu/tce/issues/1272

kcoriordan committed:
# This is a combination of 3 commits.
# This is the 1st commit message:

Removing CEIP doc #1272. Deleting unwanted file.

Signed-off-by: kcoriordan <koriordan@vmware.com>

# This is the commit message #2:

fix numbering

Signed-off-by: kcoriordan <koriordan@vmware.com>

# This is the commit message #3:

deleting unwanted file

Signed-off-by: kcoriordan <koriordan@vmware.com>

kcoriordan committed:
Removing CEIP doc #1272. Deleting unwanted file.

Signed-off-by: kcoriordan <koriordan@vmware.com>

kcoriordan committed:
Removing CEIP doc

Signed-off-by: kcoriordan <koriordan@vmware.com>

jpmcb reviewed (commented):
We've brought in the Tanzu-framework commits that have removed CEIP from the UI as part of #1314

However, #1273 is still outstanding and _may_ require a bit more work (and is unlikely to land in 0.7.0). I'd be good getting this in so that we aren't confusing users about there being an extra panel in the UI that is now gone.

@joshrosso thoughts?
